### ISSUE TITLE
Reify multiple structures

### ISSUE THREAD

**Issue by gdeest** [ku-fpg/data-reify#7]
Hi,

I'm working with cyclic dataflow graphs that can possibly have multiple outputs, and would like to reify them all at once so that the different trees share the same node labels. Problem is: right now, I can only reify the subgraphs corresponding to each output independently, resulting in different labels for the same nodes.

I think a solution would be to add the following function to data-reify:

```haskell
reifyGraphs :: (MuRef s, Traversable t) => t s -> IO (t (Graph (DeRef s)))
```

Would there be some interest in a pull request ?

At first glance, it looks like it should be pretty trivial to implement, but perhaps am I missing something.

Gaël

**Comment by andygill** [comment]
Go for it! I'd be happy to accept this.

**Comment by gdeest** [comment]
Hi, 
I submitted my attempt as PR #8. Let me know what you think !

**Comment by RyanGlScott** [comment]
Fixed by #8.
